src/eko/__init__.py

```diff
@@ -1,14 +1,21 @@
-"""Evolution Kernel Operators."""
+"""Evolution Kernel Operators.
 
+Please refer to our documentation for a full overview of the possibilities.
+"""
 from . import output, runner, version
 
+# export public constants
 __version__ = version.__version__
 
+# export public methods
+open = output.struct.EKO.open  # pylint: disable=redefined-builtin
```

Comment:
Do not redefine built-in, much better to export `EKO` top-level, and use:

```py
import EKO from eko

with EKO.open() as e:
    ...
```
that is not much different from

```py
import eko

with eko.open() as e:
    ...
```
And no one will be tempted to `import open from eko`, overwriting a built-in function.
(in general, better not to silence Pylint's warnings as much as possible)